### PR TITLE
reassigning DAG ownership to kik as per "adopt a DAG" doc

### PR DESCRIPTION
**Pull request by kik-kik**
# reassigning DAG ownership to kik as per "adopt a DAG" doc

### CODE REVIEW

File: dags/bqetl_addons.py [hunk 1]
```diff
@@ -52,7 +52,11 @@
         dataset_id="telemetry_derived",
         project_id="moz-fx-data-shared-prod",
         owner="jklukas@mozilla.com",
```
Comment (scholtzan) [inline dags/bqetl_addons.py:54]: best to remove all `jklukas` mentions here

Comment (scholtzan) [reply dags/bqetl_addons.py:54]: Everything his name touches... is yours now.

Comment (kik-kik) [reply dags/bqetl_addons.py]: yeah, starting to realise my mistake now 😓

Comment (kik-kik) [reply dags/bqetl_addons.py]: I think Jeff should be no more in this PR.

File: dags/bqetl_addons.py [hunk 2]
```diff
@@ -101,7 +101,11 @@
...
```
Comment (scholtzan) [inline dags/bqetl_addons.py:103]: One more Jeff is left here

Comment (kik-kik) [reply dags/bqetl_addons.py]: 💩  oops. Should be fixed now. Thanks for being so thorough 🙇

**Review by scholtzan — APPROVED** [review]
🎉 thanks for adopting them

**Pull request merged**